src/DI/ContainerBuilder.php

```diff
@@ -487,7 +487,7 @@ private function resolveEntityType($entity, array $recursive = []): ?string
 			if (isset($e) || ($refClass && (!$reflection->isPublic()
 				|| ($refClass->isTrait() && !$reflection->isStatic())
 			))) {
-				throw new ServiceCreationException(sprintf("Method %s() used in service '%s' is not callable.", Nette\Utils\Callback::toString($entity), $serviceName));
+				throw new ServiceCreationException(sprintf("Method %s() used in service '%s' is not callable.", Nette\Utils\Callback::toString($entity), $serviceName), isset($e) ? $e->getCode() : 0, $e ?? null);
```

Comment:
I think that code should not be copied, because it is code for different exception.